# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/DecoderTypes.h
# Commit: e903ac081227
# Full Hash: e903ac0812273c476426eba3462c4bb2977c4f2f
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 3
# Description:
#   Bug 1749047 - Move and deduplicate some utility functions to WebCodecUtils. r=chunmin
#   
#   They will be reused for the encoding side.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195699
# ==============================================================================

diff -r 029f5470314e -r e903ac081227 dom/media/webcodecs/DecoderTypes.h
--- a/dom/media/webcodecs/DecoderTypes.h	Fri Dec 22 21:39:04 2023 +0000
+++ b/dom/media/webcodecs/DecoderTypes.h	Fri Dec 22 21:39:05 2023 +0000
@@ -74,7 +74,7 @@
   static bool IsSupported(const ConfigTypeInternal& aConfig);
   static Result<UniquePtr<TrackInfo>, nsresult> CreateTrackInfo(
       const ConfigTypeInternal& aConfig);
-  static bool Validate(const ConfigType& aConfig);
+  static bool Validate(const ConfigType& aConfig, nsCString& aErrorMessage);
   static UniquePtr<ConfigTypeInternal> CreateConfigInternal(
       const ConfigType& aConfig);
   static bool IsKeyChunk(const InputType& aInput);